modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer-fragment.glsl.js

```diff
@@ -36,23 +37,30 @@ varying vec2 uv;
 void main(void) {
   geometry.uv = uv;
 
-  vec4 texColor = texture2D(iconsTexture, vTextureCoords);
-  
-  float alpha = texColor.a;
+  float alpha = texture2D(iconsTexture, vTextureCoords).a;
+
   // if enable sdf (signed distance fields)
   if (sdf) {
-    float distance = texture2D(iconsTexture, vTextureCoords).a;
-    alpha = smoothstep(buffer - vGamma, buffer + vGamma, distance);
+    alpha = smoothstep(buffer - vGamma, buffer + vGamma, alpha);
   }
 
   // Take the global opacity and the alpha from vColor into account for the alpha component
   float a = alpha * vColor.a;
-
+  
   if (a < alphaCutoff) {
-    discard;
+    if ((backgroundColor.a < 0.001) || picking_uActive) {
```

Comment:
This breaks the use case where we need to pick the text by pointing at empty pixels (e.g. the hole in "o").

I'm ok with making a breaking change in v8 to no longer support *NOT* picking the text in this case. i.e. never discard if `picking_uActive: true`.

Reply:
Can you explain?

Reply:
If you discard the background fragment in picking mode then the background won't be pickable. That makes it impossible to pick the text if your cursor is over a "hole" in the letter.

Reply:
Currently that is how TextLayer behaves in picking. It is a behavior I don't like.

Reply:
Let me try to provide a fix.